js/unit_test_runner.ts

```diff
@@ -29,6 +29,40 @@ function fmtPerms(perms: Deno.Permissions): string {
   return fmt;
 }
 
+function parseUnitTestOutput(rawOutput: Uint8Array, print: boolean) {
```

Comment:
can you add a test for this which parses `tools/testdata/unit_test_output1.txt` (and friends) 

also add return type

Reply:
Sure